### PR TITLE
feat(docs):  Starter docs 

### PR DESCRIPTION
**Pull request by yjose**
This PR is mainly to add docs and guides to help people using the starter understand our approach and use the starter effectively.
Preview link :  https://deploy-preview-25--obytes-starter.netlify.app/
If you think there are any missed sections we should add to the docs please add them in the comment section🙏

Sections:
* [x] Home page
* [x] overview
     * [x]  Motivation 
     * [x] Philosophy 
     * [x] Features
     * [x] Why expo?
     * [x] Is this stater for you?
     * [x] Stay updated 
     * [x] Contributors
     * [x] Work with us
* [x] Getting started 
* [x]  Customize app 
* [x] Project structure
* [x] Environment vars and configuration 
* [x] Rules and conventions
* [x] UI and theme configuration 
* [x] Fonts 
* [x] Components 
* [x] Forms and Inputs 
* [x] Authentication 
* [x] Data fetching 
* [ ] Storage
* [ ] Internalization 
* [x]  VsCode and snippets
* [ ]  Unit testing 
* [ ] e2e testing 
* [x] How to contribute?
* [ ] Build and distribution to stores

### CODE REVIEW

File: docs/docs/getting-started/create-new-app.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+- [Import Cost](https://marketplace.visualstudio.com/items?itemName=wix.vscode-import-cost)
+- [Cobalt 2 theme](https://marketplace.visualstudio.com/items?itemName=ahmadawais.theme-cobalt2)
+- [Turbo console log](https://marketplace.visualstudio.com/items?itemName=ChakrounAnas.turbo-console-log)
+- [i18n Ally](https://marketplace.visualstudio.com/items?itemName=lokallise.i18n-ally)
+- [Github copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot)
+
+## Running the app
+
+If the installation is done successfully, the generated app should be ready to run using the following commands in the project directory:
```
Comment (SihamBen) [inline docs/docs/getting-started/create-new-app.md:63]: I believe here we should mention that  you'll either have to install Expo Go, or setup dev env for ios and android to use simulator

File: docs/docs/getting-started/rules-and-convetions.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import PostMerge from "!!raw-loader!../../../template/.husky/post-merge";
+import CommitMsg from "!!raw-loader!../../../template/.husky/commit-msg";
+
+# Rules and Conventions
+
+In order to force a consistent code style and avoid common issues in the codebase, we have a set of rules and conventions that we follow and enforced by the starter.
+
+## Typescript
+
+This starter uses Typescript to provide type safety and to avoid common bugs in the codebase. The project configuration is based on expo config with some updates to support absolute important.
```
Comment (SihamBen) [inline docs/docs/getting-started/rules-and-convetions.mdx:17]: typo: absolute imports

File: docs/docs/getting-started/create-new-app.md (hunk 1 above)
Comment (yjose) [reply docs/docs/getting-started/create-new-app.md:63]: Nice catch, i just added more details as we can't use Expo Go with the starter

**Comment by Staceadam** [comment]
Docs looks great. Would mention 2 things

1. you might want to consider signing up for [grammarly](https://www.grammarly.com/ ) and running some of these through it
2. [search](https://docusaurus.io/docs/search) would be nice. if you don't want to go all in with Algolia there are some decent local plugins that can be hooked up [local search](https://docusaurus.io/community/resources#search)


image

**Review by SihamBen — COMMENTED** [review]
Outstanding work and effort @yjose  💯  🥇

**Pull request merged**